src/test/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapperTest.java

```diff
@@ -72,7 +72,7 @@ void mapToTranscriptionUrgencyResponses() throws Exception {
     @Test
     void mapToTranscriptionUrgencyResponse() throws Exception {
         TranscriptionUrgencyEntity transcriptionUrgencyEntity =
-            CommonTestDataUtil.createTranscriptionUrgencyEntityFromEnum(TranscriptionUrgencyEnum.OVERNIGHT);
+            CommonTestDataUtil.createTranscriptionUrgencyEntityFromEnum(TranscriptionUrgencyEnum.STANDARD);
```

Comment:
This should use one of the valid values not the old values so can be set to OVERNIGHT

Reply:
Updated